code/app/app.py

```diff
@@ -40,7 +40,7 @@ def get_config():
 AZURE_SEARCH_SERVICE = os.environ.get("AZURE_SEARCH_SERVICE")
 AZURE_SEARCH_INDEX = os.environ.get("AZURE_SEARCH_INDEX")
 AZURE_SEARCH_KEY = os.environ.get("AZURE_SEARCH_KEY")
-AZURE_SEARCH_USE_SEMANTIC_SEARCH = os.environ.get("AZURE_SEARCH_USE_SEMANTIC_SEARCH", False)
+AZURE_SEARCH_USE_SEMANTIC_SEARCH = os.environ.get("AZURE_SEARCH_USE_SEMANTIC_SEARCH", "False")
```

Comment:
Why not make this `` (empty string) which is what the EnvHelper.py is doing? More of a question why we have duplicated code - (I don't know the answer btw! :-) ) - could this function call EnvHelper?

Reply:
Good point, I've raised a new issue

https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/issues/203